metadata-ingestion/src/datahub/emitter/mcp_builder.py

```diff
@@ -90,6 +90,18 @@ class ProjectIdKey(PlatformKey):
     project_id: str
 
 
+class MetastoreKey(PlatformKey):
+    metastore: str
+
+
+class CatalogKey(MetastoreKey):
+    catalog: str
+
+
+class UnitySchemaKey(CatalogKey):
```

Comment:
I'm not a big fan of integration specific classes being added to mcp_builder.py ... we have accumulated a few and need to remove these later IMO